### DISCUSSION TITLE
Bring back stuff from xradar

### DISCUSSION THREAD

**Discussion by ghost** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1322] [General]
found some cool looking stuff in xradar and would be cool if it would be added in normal ntm mod
![2024-01-31_20 18 47](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/131010857/b7c843d6-6fa6-4d96-985e-81d4ed97fbf2)

**Comment by 9Valjew** [comment]
I stg didnt we have this conversation 1 day ago

**Reply by ghost** [reply]
what?

**Comment by HbmMods** [comment]
All of these things are just features we already have but worse, except for the stock market which was unique in how broken and nonsensical it was

**Reply by ghost** [reply]
can't you reuse those machine models and make a new machine because if that is possible it would be so cool and you could reuse the idea of ​​stock market but with bottle caps.

**Reply by 9Valjew** [reply]
he can but why should he?

**Reply by ghost** [reply]
why not?

**Reply by 9Valjew** [reply]
Because we already have these things but better, and the stock market was broken and nonsensical, did you even read bobs comment
